Makefile

```diff
@@ -329,9 +330,10 @@ release: $(GORELEASER) #EXHELP Runs goreleaser for the operator-controller. By d
 
 .PHONY: quickstart
 quickstart: export MANIFEST := https://github.com/operator-framework/operator-controller/releases/download/$(VERSION)/operator-controller.yaml
+quickstart: export DEFAULT_CATALOG := "https://github.com/operator-framework/operator-controller/releases/download/$(VERSION)/default-catalogs.yaml"
```

Comment:
We want to use the default catalog YAML released as part of the artefacts for the install script we built to users. See:

```
 $ cat install.sh 
#!/bin/bash
set -euo pipefail
IFS=$'\n\t'

olmv1_manifest=https://github.com/operator-framework/operator-controller/releases/download/v1.2.0-rc1-8-g130cb8e/operator-controller.yaml

if [[ -z "$olmv1_manifest" ]]; then
    echo "Error: Missing required MANIFEST variable"
    exit 1
fi

default_catalogs_manifest="https://github.com/operator-framework/operator-controller/releases/download/v1.2.0-rc1-8-g130cb8e/default-catalogs.yaml"
cert_mgr_version=v1.15.3
install_default_catalogs=true
```